coding-standards/elixir.md

```diff
@@ -43,6 +43,46 @@ Testing guidelines:
 
 - [Controller tests as integration tests](https://groups.google.com/g/elixir-ecto/c/BKpLf092dWs/m/VaCvfZpEBQAJ) (_pre-Phoenix Context_ but still relevant)
 
+### Test coverage
+
+Test coverage is enforced by [coveralls.io](https://coveralls.io/). Please add [excoveralls](https://github.com/parroty/excoveralls) to the list of dependencies and setup `mix.exs` file as shown here:
+
+```elixir
+def project do
+  [
+    app: :excoveralls,
```

Comment:
If this is meant to be an example maybe I'd call this app `:excoveralls_example` or `:excoveralls_demo`